combined_browser_metrics/combined_browser_metrics.model.lkml

```diff
@@ -15,6 +15,25 @@ explore: active_users_aggregates {
     sql_on: ${active_users_aggregates.country} = ${countries.code} ;;
   }
 
+  aggregate_table: rollup__period_over_period {
+    query: {
+      dimensions: [period_over_period_pivot, period_over_period_row, active_users_aggregates.app_name]
+      measures: [daily_active_users, weekly_active_users, monthly_active_users, new_profile, ad_click, organic_search_counts, search_counts, search_with_ad, uri_counts, active_hour]
+      filters: [
+        active_users_aggregates.choose_breakdown: "Month^_Day",
+        active_users_aggregates.choose_comparison: "Year",
+        active_users_aggregates.submission_date: "after 2021/01/01",
```

Comment:
The date filter must match the filter in the looks.
Looker doesn't use the aggregate when the filter is set to "this year" or when the users filter in the Explore on an  exact date within this period (for ex. "2022/02/01").